pennylane/devices/device_api.py

```diff
@@ -871,3 +958,20 @@ def eval_jaxpr(
 
         """
         raise NotImplementedError
+
+
+def _default_mcm_method(capabilities: DeviceCapabilities, shots_present: bool) -> str:
+    """Simple strategy to find the best match for the default mcm method."""
+
+    supports_one_shot = "one-shot" in capabilities.supported_mcm_methods
+    has_device_support = capabilities and "device" in capabilities.supported_mcm_methods
```

Comment:
```suggestion
    has_device_support = "device" in capabilities.supported_mcm_methods
```
Do we need this? Surely the line above failed if capabilities is empty 😅